src/module/cp/cf.py

```
@@ -220,6 +471,32 @@
+                message.reply("你还没有绑定账号，请使用 /cf bind [handle] 进行绑定")
+                return
+            if len(content) == 3 and content[2] == "finish":
+                finish_duel(message)
+            elif len(content) == 4 and content[2] == "accept":
+                accept_duel_pairing(message, content[3])
+            elif len(content) >= 4 and content[2] == "start":
+                start_duel_pairing(message, ProbInfo(
+                    tag=content[3],
+                    limit=content[4] if len(content) >= 5 and content[4] != "new" else None,
+                    newer=content[4] == "new" if len(content) == 5 else (
+                    content[5] == "new" if len(content) == 6 else False)
```

Flake8 (7.2.0)

[error] continuation line missing indentation or outdented

(E122)